examples/realTimeVideo-config/src/main.ts

```diff
@@ -0,0 +1,94 @@
+import { Authentication, Fleet, KeyValue } from "@formant/data-sdk";
+import "@formant/ui-sdk-joystick";
+import "@formant/ui-sdk-realtime-player";
+import { RealtimePlayer } from "@formant/ui-sdk-realtime-player";
+import "./style.css";
+
+const loadVideo = async (defaultCamera: number) => {
+  (el("formant-realtime-player") as RealtimePlayer).drawer.start();
+  let indicator = document.getElementById("loading-indicator");
+  try {
+    if (await Authentication.waitTilAuthenticated()) {
+      const device = await Fleet.getCurrentDevice();
+
+      await device.startRealtimeConnection();
+      device.addRealtimeListener((_peerId: any, message: any) => {
+        (el("formant-realtime-player") as RealtimePlayer).drawVideoFrame(
+          message.payload.h264VideoFrame
+        );
+      });
+      let videoStreams = await device.getRealtimeVideoStreams();
+      let settingsIcon = document.getElementById("settings");
+      let settingsBlock = document.getElementById("video-settings");
+
+      settingsIcon!.style.display = "flex";
+      settingsIcon?.addEventListener("click", () => {
+        settingsBlock!.style.display = "flex";
+      });
+      renderCameraOptions(videoStreams);
+
+      device.startListeningToRealtimeVideo(videoStreams[defaultCamera]);
+      setTimeout(() => {
+        indicator!.style.display = "none";
+      }, 3000);
+      el("formant-realtime-player").style.display = "block";
+    }
+  } catch (e) {
+    log((e as Error).message);
+  }
+};
+
+function saveDefaultCamera() {
+  const videoSource = document.querySelectorAll("p.camera-option");
+  videoSource.forEach((video) => {
+    video.addEventListener("click", (event) => {
+      const camera = (event.target as HTMLParagraphElement).id;
+      KeyValue.set("defaultVideoStream", camera.at(-1)!);
+      let settingsBlock = document.getElementById("video-settings");
+      settingsBlock!.style.display = "none";
+      location.reload();
+    });
+  });
+}
+
+function renderCameraOptions(videoStreams: any[]) {
+  videoStreams.forEach((_, idx) => {
+    $(".camera-options").append(
+      `<p id=${`video-${idx}`} class="camera-option">${_.name}</p>`
+    );
+  });
+}
+
+function log(msg: string) {
+  el("#log").innerHTML = msg + "<br>" + el("#log").innerHTML;
+}
+
+function el(selector: string) {
+  return document.querySelector(selector) as HTMLElement;
+}
```

Comment:
It might be worth being _slightly_ more defensive in here, rather than just _blindly casting_ to HTMLElement, you might get slightly better debugging info if you assert the found element (or `null`) _is_ an HTMLElement instead:

```tsx
function el(selector: string): HTMLElement {
  const match = document.querySelector(selector);
  if (!(match instanceof HTMLElement)) { throw new Error(`No element found: "${selector}"`);  }
  return match;
}
```

thats _if_ you _really_ need it to be a `HTMLElement` and not just its parent class `Element`. If you want the _latter_ and just want to assert that the `querySelector` didn't return `null`, then you can use the same trailing non-null assertion, as before:

```ts
const el = (selector:string) => document.querySelector(selector)!;
```